drivers/mpsl/clock_control/Kconfig

```diff
@@ -24,6 +24,14 @@ config CLOCK_CONTROL_NRF_K32SRC_XTAL
 config CLOCK_CONTROL_NRF_K32SRC_SYNTH
 	bool "Synthesized from HFCLK"
 
+config CLOCK_CONTROL_NRF_K32SRC_EXT_LOW_SWING
+	depends on SOC_SERIES_NRF52X
+	bool "External low swing"
```

Comment:
is this descriptive enough for our users to know what it means ?
Also, please add a help text with a more descriptive text field.

Reply:
This was taken directly from Zephyr: https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/clock_control/Kconfig.nrf#L40. Therefore I believe we should keep it as is.
As a reference, users of MPSL will find more detailed documentation in the MPSL API header: https://github.com/NordicPlayground/nrfxlib/blob/master/mpsl/include/mpsl_clock.h#L33

Reply:
I don't necessarily agree, as this could indicate this ought to change upstream as well.
but as it is used upstream, changing it now would be outside scope of this PR.


@nordic-krch please take a look at this comment:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1974/commits/f4e39715fac385b174234632559906a8c571a6da#r387663022
and consider if we should improve the description, to make it easier for out users.
https://github.com/zephyrproject-rtos/zephyr/commit/c661cc6c781aaace5759921bdb570d552618492c

Reply:
@nordic-krch If you do update it, please do it for both.